_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -285,13 +285,13 @@ This page has an undefined [default language][default page language] because it
 		<title>Paul put dire comment on tape</title>
 	</head>
 	<body>
-		<p>Paul put dire comment on tape</p>
+		<p>Paul put dire comment on tape.</p>
 	</body>
 </html>
 ```
 
 [valid language tag]: #valid-language-tag
-[default page language]: #default-page-language
+[default page language]: #default-element-language
```

Comment:
Why this change in this rule?

Reply:
I've mostly renamed the definition +adapt it to make it work on any element (not just `html`), hence propagating the new name. This is actually the same definition (or so I hope 🤞).

Git doesn't see the rename anymore (sees a delete+create instead) because it ends up with a bit too many changes in its opinion…